include/libc/math.h

```diff
@@ -1,10 +1,11 @@
 #ifndef LIBC_MATH_H
 #define LIBC_MATH_H
 
-#define M_PI 3.14159265358979323846f
-#define M_SQRT2 1.41421356237309504880f
-#define M_SQRT1_2 0.70710678118654752440f	/* 1/sqrt(2) */
-#define FLT_MAX 340282346638528859811704183484516925440.0f
+#define F_PI 3.14159265358979323846f
+#define F_SQRT2 1.41421356237309504880f
+#define F_SQRT1_2 0.70710678118654752440f /* 1/sqrt(2) */
+
+#define MAXFLOAT 3.40282347e+38f
```

Comment:
I think we already had the `FLT_MAX` vs `MAXFLOAT` discussion before.
iirc we decided to use `FLT_MAX` because it is an standard macro while `MAXFLOAT` isn't
https://en.cppreference.com/w/c/types/limits

Reply:
Ahh, didn't know we had already discussed it in MM already. I can revert it back later.

Reply:
Out of curiosity, should I also remove `SHT_MINV` and also rename the Max to `SHRT_MAX` to match the page you linked?

Reply:
Actually I'm trying to dig for the discussion and I can't find it. We may have discussed something else related to the C standard.

I kinda prefer using the C standard names, but I'm open to renaming those constants